framework/src/transfers/MultiAppConservativeTransfer.C

```diff
@@ -256,7 +255,6 @@ MultiAppConservativeTransfer::postExecute()
             EXEC_TRANSFER, Moose::POST_AUX, _to_postprocessors_to_be_preserved[0]);
     }
 
-    _console << "Finished Conservative transfers " << name() << std::endl;
```

Comment:
I do not think I like this now but I can be convinced if there is a strong reason.  We break all the encapsulations

Reply:
@GiudGiud had a comment on this as well. I hope you can be convinced with the comparison. Note that individual transfers can still print, just do not need to have this begin-finish two lines of prints.